lib/shower/default-options.js

```diff
@@ -1,17 +1,13 @@
 export default {
     container_selector: '.shower',
-
-    debug_mode: false,
-    debug_mode_classname: 'debug',
-
-    sessionstore_key: 'shower',
-
-    slides_selector: '.shower .slide',
+    slides_selector: '.slide',
```

Comment:
`slides_selector` will be queried inside the container, so we can remove `container_selector` from it.